From a0c61409f96b47f759e795cf0b8dcf66e5eb7a75 Mon Sep 17 00:00:00 2001
From: Steven Webster <steven.webster@windriver.com>
Date: Wed, 14 Sep 2022 10:39:45 -0400
Subject: [PATCH] Fix whitespace chomping in services manifest

Currently, the services template chomps whitespace
(including the newline character) at the end of the
EnableGPS conditional.

When rendered, this results in something like:

  ports:
    - protocol: TCP
      port: 2497
      targetPort: 2497---
apiVersion: v1
kind: Service
metadata:

Note: this was seen using helm v3.2.1

Signed-off-by: Steven Webster <steven.webster@windriver.com>
---
 charts/sts-silicom/templates/services.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/charts/sts-silicom/templates/services.yaml b/charts/sts-silicom/templates/services.yaml
index 926cc56..94229c9 100644
--- a/charts/sts-silicom/templates/services.yaml
+++ b/charts/sts-silicom/templates/services.yaml
@@ -14,7 +14,7 @@ spec:
     - protocol: TCP
       port: {{.Values.GpsSvcPort}}
       targetPort: {{.Values.GpsSvcPort}}
-{{- end -}}
+{{- end }}
 ---
 apiVersion: v1
 kind: Service
-- 
2.25.1

